src/did-change-watched-files.jl

```diff
@@ -0,0 +1,93 @@
+using TOML
+
+const DID_CHANGE_WATCHED_FILES_REGISTRATION_ID = "jetls-did-change-watched-files"
+const DID_CHANGE_WATCHED_FILES_REGISTRATION_METHOD = "workspace/didChangeWatchedFiles"
+
+function did_change_watched_files_registration()
+    Registration(;
+        id = DID_CHANGE_WATCHED_FILES_REGISTRATION_ID,
+        method = DID_CHANGE_WATCHED_FILES_REGISTRATION_METHOD,
+        registerOptions = DidChangeWatchedFilesRegistrationOptions(;
+            watchers = [
+                FileSystemWatcher(;
+                    globPattern = "**/JETLSConfig.toml",
```

Comment:
How about renaming `JETLSConfig.toml` to `.JETLSConfig.toml`? I don't think there is a standard of how to name configurations files in our ecosystem, but at least we have `.JuliaFormatter.toml`.

Reply:
It seems better for me. I'll make the change :+1: